src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/health/health.component.ts

```diff
@@ -32,44 +73,63 @@ export class HealthComponent implements OnInit, OnDestroy {
     });
   }
 
+  prepareReadWriteRatio(chart, data) {
+    const ratioLabels = [];
+    const ratioData = [];
+
```

Comment:
The separation is for the sake of readability, no need for a logical separation (another method).

Reply:
If this were python I would agree because 2 lines separate each method if you are following pep8. But as we use in all other typescript files 1 line to separate each method, it looks to me like a new method and this method as a hole isn't that complicated so no separation or comment should be needed.

Please remove the separations.

Reply:
This can be addressed post-merge.